exercises/javascript-timers/interval.js

```diff
@@ -0,0 +1,19 @@
+var $h1 = document.querySelector('h1');
+
+function countdown() {
+  if ($h1.textContent === '4') {
+    $h1.textContent = '3';
+
+  } else if ($h1.textContent === '3') {
+    $h1.textContent = '2';
+
+  } else if ($h1.textContent === '2') {
+    $h1.textContent = '1';
+
+  } else if ($h1.textContent === '1') {
+    $h1.textContent = '~Earth Beeeelooowww Us~!';
+    clearInterval(countdownFrom4);
+  }
```

Comment:
This is very manual. What if you want to make he countdown longer? Add an additional conditional for each number? You may want to make a counter variable which you can use to keep track of what the next number is, and if it is zero, you should set the text content to '~Earth Beeeelooowww Us~!'